packages/angular/migrations.json

```diff
@@ -739,6 +739,86 @@
     "12.8.0": {
       "version": "12.8.0",
       "packages": {
+        "@angular/cli": {
+          "version": "~12.2.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular-devkit/build-angular": {
+          "version": "~12.2.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular-devkit/build-optimizer": {
+          "version": "^0.1202.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular/core": {
+          "version": "^12.2.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular/common": {
+          "version": "^12.2.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular/forms": {
+          "version": "^12.2.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular/elements": {
+          "version": "^12.2.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular/compiler": {
+          "version": "^12.2.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular/compiler-cli": {
+          "version": "^12.2.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular/platform-browser": {
```

Comment:
Can you add `@angular/platform-server` please? I've noticed it was missing in previous migrations.